spec/lib/dor/update_marc_record_service_spec.rb

```diff
@@ -273,12 +273,26 @@
       content_metadata_ng_xml = Nokogiri::XML(build_content_metadata_5)
       content_metadata_ds = double(Dor::ContentMetadataDS)
 
-      expect(d).to receive(:id).and_return('')
+      expect(d).to receive(:id).and_return('aa111aa2222')
       expect(d).to receive(:datastreams).exactly(4).times.and_return({'contentMetadata' => content_metadata_ds})
       expect(content_metadata_ds).to receive(:ng_xml).twice.and_return(content_metadata_ng_xml)
 
       updater = Dor::UpdateMarcRecordService.new(d)
-      expect(updater.file_id).to eq('|xfile:wt183gy6220_00_0002.jp2')
+      expect(updater.file_id).to eq('|xfile:aa111aa2222%2Fwt183gy6220_00_0002.jp2')
+    end
+
+    # Added thumb based upon recommendation from Lynn McRae for future use
+    it 'should return correct file_id from a valid contentMetadata with resource type = thumb' do
+      d = double(Dor::Item)
+      content_metadata_ng_xml = Nokogiri::XML(build_content_metadata_6)
+      content_metadata_ds = double(Dor::ContentMetadataDS)
+
+      expect(d).to receive(:id).and_return('bb111bb2222')
+      expect(d).to receive(:datastreams).exactly(4).times.and_return({'contentMetadata' => content_metadata_ds})
```

Comment:
lol @ complaining about redundant curly braces AND missing the space inside the redundant curly braces.  WHICH ONE IS IT HOUND!?!?!?